agents-api/agents_api/app.py

```diff
@@ -11,22 +13,23 @@
 from .env import api_prefix, hostname, protocol, public_port
 
 
-class Assignable(Protocol):
-    def __setattr__(self, name: str, value: Any) -> None: ...
+class State(Protocol):
```

Comment:
What's the difference between this `State` class and the one in `agents-api/agents_api/activities/container.py`?

Reply:
i.e. can't we have only one of them?